playwright/tests/syncTabs.spec.ts

```diff
@@ -0,0 +1,134 @@
+import { test, expect, Page } from '@playwright/test'
+import { mockApi } from '../utils/mockApi'
+import { warnSlowApi } from '../utils/warnSlowApi'
+import { expectNoFatal } from '../utils/expectNoFatal'
+import { privateKey, password, privateKeyAddress } from '../utils/test-inputs'
+import { addPersistedStorage, clearPersistedStorage } from '../utils/storage'
+
+test.beforeEach(async ({ context, page }) => {
+  await warnSlowApi(context)
+  await mockApi(context, 0)
+  await clearPersistedStorage(page)
+})
+
+test.afterEach(async ({ context }, testInfo) => {
+  await expectNoFatal(context, testInfo)
+})
+
+test.describe('syncTabs', () => {
+  test.describe('lock second tab after locking wallet', () => {
+    test('unpersisted', async ({ page, context }) => {
+      await page.goto('/open-wallet/private-key')
+      await page.getByPlaceholder('Enter your private key here').fill(privateKey)
+      await page.keyboard.press('Enter')
+      await expect(page.getByText('Loading account')).toBeVisible()
+      await expect(page.getByText('Loading account')).toBeHidden()
+      await expect(page.getByTestId('account-selector')).toBeVisible()
+      await expect(page).toHaveURL(new RegExp(`/account/${privateKeyAddress}`))
+
+      const tab2 = await context.newPage()
+      await testLockingIsSynced(page, tab2)
+    })
+
+    test('persisted', async ({ page, context }) => {
+      await page.goto('/open-wallet/private-key')
+      await page.getByPlaceholder('Enter your private key here').fill(privateKey)
+      await page.getByText('Store private keys locally, protected by a password').check()
+      await page.getByPlaceholder('Enter your password here').fill(password)
+      await page.getByPlaceholder('Re-enter your password').fill(password)
+      await page.keyboard.press('Enter')
+      await expect(page.getByText('Loading account')).toBeVisible()
+      await expect(page.getByText('Loading account')).toBeHidden()
+      await expect(page.getByTestId('account-selector')).toBeVisible()
+      await expect(page).toHaveURL(new RegExp(`/account/${privateKeyAddress}`))
+
+      const tab2 = await context.newPage()
+      await testLockingIsSynced(page, tab2)
+      await expect(page.getByRole('button', { name: 'Unlock' })).toBeVisible()
+      await expect(tab2.getByRole('button', { name: 'Unlock' })).toBeVisible()
+    })
+
+    test('incognito', async ({ page, context }) => {
+      await addPersistedStorage(page)
+      await page.goto('/')
+      await page.getByRole('button', { name: 'Continue without the profile' }).click()
+      await expect(page.getByTestId('account-selector')).toBeHidden()
+
+      const tab2 = await context.newPage()
+      await tab2.goto('/')
+      await expect(tab2.getByRole('button', { name: 'Unlock' })).toBeHidden()
```

Comment:
This is failing on CI, but not locally

Reply:
I haven't found a reason why it matches differently on CI, only fixed it with https://github.com/oasisprotocol/oasis-wallet-web/commit/77be85730f689ec2e9586c0a4c399c2b6fbe5ce7.
`await expect(tab2.getByRole('button', { name: 'Unlock' })).toBeHidden()` on CI matched
```tsx
<a aria-label="Open wallet">
  <button tabindex="-1">
    <div>
      <svg aria-label="Unlock">..</svg>
      Open wallet
</
```